src/stage1/migrate_info.rs

```diff
@@ -241,6 +256,7 @@ impl MigrateInfo {
             nwmgr_files,
             system_proxy_files,
             backup,
+            change_dt_to: opts.change_dt_to().to_owned().unwrap_or("".to_string()),
```

Comment:
If I followed it right, this value ends up in the stage 2 config. So, when dealing with this, stage 2 will need to check if the value is an empty string or a valid DT. I think using an `Option` would be clearer ("if the value is provided, it is a DT for sure") and more idiomatic.

Reply:
Yes, still WIP :)

I realised i don't need it in MigrationInfo, but only need to carry the information into Stage2, so I simplified it a bit